app/models/post.rb

```diff
@@ -0,0 +1,30 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  validates :title, presence: true
+  validates :title, length: { maximum: 250 }
+  validates :comments_counter, comparison: { greater_than_or_equal_to: 0 }
+  validates :likes_counter, comparison: { greater_than_or_equal_to: 0 }
```

Comment:
- [ ] Kindly validate comments_counter and likes_counter to be of type integer too. This is demanded by the project requirements.